docs/docs/reference/release-notes/migrating-source-plugin-from-v3-to-v4.md

```diff
@@ -0,0 +1,276 @@
+# Upgrade your source plugins for Gatsby 4
+
+Gatsby 4 is here! Following on the heels of Gatsby 3, Gatsby 4 further improves build performance and introduces new parallel processing capabilities. In the guide below, we'll walk you through preparing your source plugin for Gatsby 4.
```

Comment:
Do we want to mention anything about it being in Alpha here or just talk about v4 generally?